cg/store/api/add.py

```diff
@@ -34,10 +34,16 @@
 class AddHandler(BaseHandler):
     """Methods related to adding new data to the store."""
 
-    def generate_unique_petname(self) -> str:
+    def generate_readable_sample_id(self) -> str:
         while True:
```

Comment:
Can we have a docstring here, for user comprehension, that this is not part of the typical flow to give sample ids petnames. I was extremely confused by this just now

Reply:
I don't have any insight as to what the normal flow is, me touching this function is pretty accidental. I'm guessing sample id:s are usually created in lims? Do you have a suggestion for a dcostring?

Reply:
I think the tricky part here is that it would set a petname for the sample internal id, which we apparently sometimes do in development. 

```suggestion
    def generate_readable_sample_id(self) -> str:
    """Generates a petname as sample internal id for develepment purposes. Not used in normal production flow."""
        while True:
```

Something along these lines so in future this wont make it into the flow in some way.